quix-frontend/client/src/lib/sql-autocomplete/adapter/tests/test-utils/tests-utils.ts

```diff
@@ -13,9 +15,89 @@ export const runAdapterGetCompletersTest = (
 ) => {
   const config = new MockDbInfoService();
   const autocomp = new SqlAutocompleter(config);
-
+if (testnumber < 18) {
   it(`test #${testnumber} should return comleters = ${expected}`, async () => {
     const completers = await autocomp.getCompletionItemsFromQueryContext(queryContext);
     expect(completers).to.be.deep.equal(expected);
   });
+}
+else {
+  queryContext = {
```

Comment:
I don't understand what are you doing here..
you already got the queryContext in test input.. use it!